pkgs/development/libraries/libopencore/default.nix

```diff
@@ -0,0 +1,16 @@
+{stdenv, fetchurl}:
+
+stdenv.mkDerivation {
+  name = "opencore-amr";
```

Comment:
The name should contain the version in it. Usually something like
```
let
  version = "0.1.5";
in
stdenv.mkDerivation {
  name = "opencore-amr-${version}";
```

would be done, also adding the version to the url component in the same way.